dotnet/src/SemanticKernel.Abstractions/Services/AIServiceExtensions.cs

```diff
@@ -91,19 +91,19 @@ public static class AIServiceExtensions
             return (service, settings);
         }
 
-        var message = new StringBuilder($"Required service of type {typeof(T)} not registered.");
+        var message = new StringBuilder().Append("Required service of type ").Append(typeof(T)).Append(" not registered.");
```

Comment:
nit.

Wonder if in a future .NET release all interpolated strings could be compiled as string builders, if that makes sense.

Reply:
https://devblogs.microsoft.com/dotnet/string-interpolation-in-c-10-and-net-6/